unittest/test_solvers.cpp

```diff
@@ -68,6 +68,9 @@ void test_solver_against_kkt_solver(SolverTypes::Type solver_type, ActionModelTy
   boost::shared_ptr<crocoddyl::SolverAbstract> solver =
       boost::static_pointer_cast<crocoddyl::SolverKKT>(solver_factory.create(solver_type, action_type, T));
 
+  //boost::shared_ptr<crocoddyl::SolverAbstract> solverMultipleActions =
```

Comment:
Is there a proposed to keep this commented lines?

Reply:
No, I've removed this comment.